### ISSUE TITLE
[Bug] some metrics are not prefixed with "temporal_"

### ISSUE THREAD

**Issue by antmendoza** [temporalio/sdk-typescript#1353]
### Describe the bug

Some metrics emited by the sdk are not prefixed with "temporal_", concretely `request` and `long_request`  

> url=http://localhost:9090/
> curl -s $url/api/v1/label/name/values | jq -r ".data[]" | sort
 
long_request
long_request_latency_bucket
long_request_latency_count
long_request_latency_sum
request
request_latency_bucket
request_latency_count
request_latency_sum
temporal_activity_execution_latency_bucket
temporal_activity_execution_latency_count
temporal_activity_execution_latency_sum
temporal_activity_poll_no_task
temporal_activity_schedule_to_start_latency_bucket
temporal_activity_schedule_to_start_latency_count
temporal_activity_schedule_to_start_latency_sum
temporal_activity_task_received
temporal_num_pollers
temporal_sticky_cache_hit
temporal_sticky_cache_size
temporal_worker_start
temporal_worker_task_slots_available
temporal_workflow_completed
temporal_workflow_endtoend_latency_bucket
temporal_workflow_endtoend_latency_count
temporal_workflow_endtoend_latency_sum
temporal_workflow_task_execution_latency_bucket
temporal_workflow_task_execution_latency_count
temporal_workflow_task_execution_latency_sum
temporal_workflow_task_queue_poll_empty
temporal_workflow_task_queue_poll_succeed
temporal_workflow_task_replay_latency_bucket
temporal_workflow_task_replay_latency_count
temporal_workflow_task_replay_latency_sum
temporal_workflow_task_schedule_to_start_latency_bucket
temporal_workflow_task_schedule_to_start_latency_count
temporal_workflow_task_schedule_to_start_latency_sum

This happen with v.1.9.0 and v.1.9.1, v1.8.6 seems ok 

### Environment/Versions

- OS and processor: [e.g. M1 Mac, x86 Windows, Linux]
- Temporal Version: [e.g. 1.14.0?] and/or SDK version
- Are you using Docker or Kubernetes or building Temporal from source?

**Comment by paymog** [comment]
I'm also seeing a metric called `target_info` which doesn't have that prefix

```
# HELP long_request Count of long-poll request successes by rpc name
# HELP long_request_failure Count of long-poll request failures by rpc name
# HELP long_request_latency Histogram of client long-poll request latencies
# HELP request Count of client request successes by rpc name
# HELP request_latency Histogram of client request latencies
# HELP target_info Target metadata
# HELP temporal_activity_execution_latency Histogram of activity execution latencies
# HELP temporal_activity_poll_no_task Count of activity task queue poll timeouts (no new task)
# HELP temporal_activity_schedule_to_start_latency Histogram of activity schedule-to-start latencies
# HELP temporal_activity_task_received Count of activity task queue poll successes
# HELP temporal_num_pollers Current number of active pollers per queue type
# HELP temporal_sticky_cache_hit Count of times the workflow cache was used for a new workflow task
# HELP temporal_sticky_cache_size Current number of cached workflows
# HELP temporal_worker_start Count of the number of initialized workers
# HELP temporal_worker_task_slots_available Current number of available slots per task type
# HELP temporal_workflow_completed Count of successfully completed workflows
# HELP temporal_workflow_continue_as_new Count of continued-as-new workflows
# HELP temporal_workflow_endtoend_latency Histogram of total workflow execution latencies
# HELP temporal_workflow_failed Count of failed workflows
# HELP temporal_workflow_task_execution_latency Histogram of workflow task execution (not replay) latencies
# HELP temporal_workflow_task_queue_poll_empty Count of workflow task queue poll timeouts (no new task)
# HELP temporal_workflow_task_queue_poll_succeed Count of workflow task queue poll successes
# HELP temporal_workflow_task_replay_latency Histogram of workflow task replay latencies
# HELP temporal_workflow_task_schedule_to_start_latency Histogram of workflow task schedule-to-start latencies
```

**Comment by mjameswh** [comment]
@antmendoza Would you mind checking if Python has the same issue?

**Comment by antmendoza** [comment]
@mjameswh the current prometheus example https://github.com/temporalio/samples-python/tree/main/prometheus shows all metrics with temporal_ prefix 

it uses [python-sdk 1.5.0](https://github.com/temporalio/sdk-python/releases/tag/1.5.0) 

https://github.com/temporalio/samples-python/blob/6bcab28f85461c13e594ebc15ec3a111533a3ae7/pyproject.toml#L20

**Comment by mjameswh** [comment]
Fixed by #1441.
